e2e/node/manager_test.go

```diff
@@ -325,7 +325,7 @@ func (s *ManagerTestSuite) TestRestartNode() {
 	// make sure we can read the first byte, and it is valid (for Rinkeby)
 	firstHash, err := e2e.FirstBlockHash(s.NodeManager)
 	s.NoError(err)
-	s.Equal("0x6341fd3daf94b748c72ced5a5b26028f2474f5f00d824504e4fa37a75767e177", firstHash)
+	s.Equal("0x28c4da1cca48d0107ea5ea29a40ac15fca86899c52d02309fa12ea39b86d219c", firstHash)
```

Comment:
Head hashes are still wip, right?

Reply:
I myself am unsure here, in my mind we would need to have some means of pass in expected hash for validation in test else on test that we are able to retrieve a particular network's giving hash.

Reply:
For now, I think a test helper returning an expected hash is enough. We only need 3 hashes (Rinkeby, Ropsten, StatusChain) and they're rarely modified or added.